src/components/calcite-modal/calcite-modal.tsx

```diff
@@ -98,6 +98,10 @@ export class CalciteModal {
     }
   }
 
+  disconnectedCallback(): void {
+    this.removeOverflowHiddenClass;
```

Comment:
Don't you have to call this like a method?

Reply:
ah yeah you do.

Also, can you add a test for this?